src/app-layout/drawer.tsx

```diff
@@ -63,24 +57,15 @@ export function Drawer({
   ariaLabels,
   children,
   isOpen,
-  isHidden,
   isMobile,
-  hasDividerWithSplitPanel,
   onToggle,
   onClick,
   onLoseFocus,
-  extendRight = 0,
```

Comment:
This value is never set and this makes `styles['opaque-background']` never applied. Removing